server/controllers/chartsController.js

```diff
@@ -2,6 +2,8 @@ import { ObjectID } from 'mongodb'
 
 import { collections } from '../utils/mongoCollections'
 
+const Totals = 'Totals'
```

Comment:
can we be a little more descriptive here and also use all caps for the constant?

```suggestion
const TOTALS_STUDY = 'Totals'
```

Reply:
https://github.com/AMP-SCZ/dpdash/pull/137/commits/f87a63ed70a41555dabd1aeb8becb444ca08a66c